### PR TITLE
User can add types of events they're interested in to their profile

### PR DESCRIPTION
**Pull request by lymichelle21**
New Stuff:
- User can add a bio and the types of events they're interested in through a multi-select alert dialog
- User bio and interested event categories are displayed in the profile fragment
- User can see animated transitions when logging in, signing up, and adding a photo to a photo album

Screenshots:

Screen Shot 2022-07-13 at 10 40 51 AM

Screen Shot 2022-07-13 at 10 41 19 AM

Screen Shot 2022-07-13 at 10 41 34 AM

### CODE REVIEW

File: app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java [hunk 1]
```diff
@@ -26,16 +36,76 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
+    }
+
     private void signUpUser() {
         ParseUser user = new ParseUser();
         user.setUsername(etUsername.getText().toString());
         user.setPassword(etPassword.getText().toString());
         user.put("zip", etZip.getText().toString());
+        user.put("bio", etBio.getText().toString());
+        if (categoryListOfStrings == null) {
+            Toast.makeText(SignUpActivity.this, "Error: Sign up failed", Toast.LENGTH_LONG).show();
```
Comment (erikweissfb) [inline app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java:104]: 💡 Usually if a failure is due to the user, we want to clearly communicate what they can do to address the error (in this case, ensure they select at least one category).

**Review by erikweissfb — APPROVED** [review]

File: app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java [hunk 2]
```diff
@@ -26,16 +36,76 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
         Button btnSignUp = findViewById(R.id.btnSignUp);
 
+        setUpEventCategoryDropdown();
         btnSignUp.setOnClickListener(v -> signUpUser());
     }
 
+    private void setUpEventCategoryDropdown() {
+        boolean[] selectedCategories;
+        ArrayList<Integer> categoryList = new ArrayList<>();
+        String[] categoryArray = {"music", "visual-arts", "performing-arts", "film",
```
[the remark this answers is not in the document]
Comment (lymichelle21) [reply app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java:50]: Will look into this! I think I could display the categories in a more user-friendly way, and I have some ideas on how to implement this. I like the idea of the category endpoint, but it only returns Yelp business categories while I need the event categories only: https://www.yelp.com/developers/documentation/v3/event_categories_list

[the remark this answers is not in the document]
Comment (erikweissfb) [reply app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java:50]: Ah thanks for looking into it, I didn't know they were a different set of categories!

File: app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java (hunk 1 above)
Comment (lymichelle21) [reply app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java:104]: Will fix this!

**Pull request merged**